Sources/Bow/Data/Validated.swift

```diff
@@ -14,9 +14,9 @@ public typealias ValidatedNEA<E, A> = Validated<NEA<E>, A>
 
 /// Validated is a data type to represent valid and invalid values. It is similar to `Either`, but with error accumulation in the invalid case.
 public final class Validated<E, A>: ValidatedOf<E, A> {
-    private let value: _Validated<E, A>
+    fileprivate let value: Either<E, A>
```

Comment:
Although it removes some sort of duplication and it is just used internally, I think it is convenient to maintain the `valid`/`invalid` cases in Validated.

Reply:
I understand, will revert.